pkg/statistics/histogram.go

```diff
@@ -1443,6 +1443,9 @@ func MergePartitionHist2GlobalHist(sc *stmtctx.StatementContext, hists []*Histog
 	if expBucketNumber == 0 {
 		return nil, errors.Errorf("expBucketNumber can not be zero")
 	}
+	if len(hists) == 0 {
```

Comment:
```suggestion
	// This only happens when there are no records for the histogram in the system. It may be caused by the DDL event not having been handled yet.
	if len(hists) == 0 {
```